app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_list = command.split(" ")
```

Comment:
The task checklist specifies that you should ensure the input string contains exactly three elements separated by spaces, with 'cp' as the first element. Consider adding a check for this condition.